packages/region/src/models/country.ts

```diff
@@ -30,13 +30,15 @@ export default class Country {
   @Property({ columnType: "text" })
   display_name: string
 
+  @Property({ columnType: "text", nullable: true })
+  region_id?: string | null = null
```

Comment:
The ? Is not required to be here, since there is a default initializer it is by default optional

Reply:
You're right. More generally, this should not be a blocking comment (I know you haven't done that, but just thought I'd point it out)